tests/drivers/at/main.c

```diff
@@ -285,6 +285,67 @@ static int remove_urc(int argc, char **argv)
 }
 #endif
 
+static int sneaky_urc(int argc, char **argv)
```

Comment:
how specific is this test? Is this just one ublox thing or this  LTE modem quiet general?

Reply:
should this be switchable?

Reply:
not compile it in if not needed (but this was because i didn't realise that sneaky_urc.py simulates a modem